Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -246,11 +259,10 @@ inline HRESULT GetTarget(ID2D1Image** pTarget) {
     // See the comments above _CGContextSetShadowProjectionTransform.
     CGAffineTransform shadowProjectionTransform{ CGAffineTransformIdentity };
 
-    // TODO(DH) GH#1070 evaluate these defaults; they should be set by context creators.
-    bool allowsAntialiasing = false;
-    bool allowsFontSmoothing = false;
-    bool allowsFontSubpixelPositioning = false;
-    bool allowsFontSubpixelQuantization = false;
+    _CGAntialiasToggle allowsAntialiasing = _kCGToggleDefault;
```

Comment:
These do not have a direct mapping to D2D. The default should be _true, allow_. They aren't trimodal.